### DISCUSSION TITLE
Inserting my notes

### DISCUSSION THREAD

**Discussion by DJHL** [bramses/chatgpt-md#32] [Q&A] [answered]
After testing the plugin, it seems that it sends all previous questions and answers again to the API when I ask a new question? At least, that's my impression because there is an effect when I delete or change previous parts.

My question: where and how can I enter my own notes in the markdown document? Without sending them to the API.

**Comment by bramses** [comment, marked as answer]
Hi @DJHL, so ChatGPT MD works by splitting a markdown note into messages and sends that directly to OpenAI in their spec: 

```
[{ role: "", content: ""},...]
```

If you want to have a scratchpad of stuff you want to write that you don't send to the api I'd recommend using another note and then add a back link to the chat. You can also highlight text and create a new chat from that.
